charts/datafold/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: datafold
 description: Helm chart package to deploy Datafold on kubernetes.
 type: application
-version: 0.6.46
+version: 0.6.47
```

Comment:
You should also modify the version for helm-charts in cloud-infra

Reply:
thanks!

https://github.com/datafold/cloud-infra/pull/836